### PR TITLE
Add Sri Lanka configuration

### PR DESCRIPTION
**Pull request by wadhwamatic**

### CODE REVIEW

**Review by JorgeMartinezG — APPROVED** [review]
Tested locally

image

File: src/config/srilanka/tables.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        "type": "line",
+        "stacked": false,
+        "transpose": true,
+        "category": "Month",
+        "data": "data"
+      }
+    },
+    "poverty": {
+      "title": "Poverty headcount 2016 vs 2018",
+      "table": "./data/tables/NSO_Poverty_Headcount_Admin1.csv",
```
Comment (ericboucher) [inline src/config/srilanka/tables.json:28]: Are these tables in there already and relevant to sri lanka?

Comment (wadhwamatic) [reply src/config/srilanka/tables.json:28]: no, but it seems we require tables.json to have content. any suggestions on how to manage deployments without tables?

Comment (ericboucher) [reply src/config/srilanka/tables.json:28]: Linked to #373

**Review by ericboucher — APPROVED** [review]

**Pull request merged**